crates/rust-analyzer/src/conv.rs

```diff
@@ -343,7 +343,7 @@ impl Conv for HighlightTag {
             }
 
             HighlightTag::LITERAL_STRING => SemanticTokenType::STRING,
-            HighlightTag::LITERAL_ATTRIBUTE => SemanticTokenType::KEYWORD,
+            HighlightTag::LITERAL_ATTRIBUTE => "attribute".into(),
```

Comment:
Did you mean to commit this? I think it's causing some issues since we do not have an attribute token type and the `conv` below is failing.

Reply:
Opus, no